docs/integrations/llms/llama-index.md

```diff
@@ -0,0 +1,70 @@
+---
+integration: logfire
+---
+
+**Logfire** supports instrumenting calls to different LLMs with one extra line of code.
+Since LlamaIndex supports multiple LLMs, you can use **Logfire** to instrument calls to those LLMs.
```

Comment:
```suggestion
The way we recommend instrumenting LlamaIndex is to instrument the underlying LLM, and to rely on that instrumentation to ensure that calls made by LlamaIndex end up in Logfire.

**Logfire** supports instrumenting calls to different LLMs with one extra line of code.
Since LlamaIndex supports multiple LLMs, you can use **Logfire** to instrument calls to those LLMs.
```

I think we can be a bit more explicit here that we aren't specifically instrumenting LlamaIndex, we are just relying on its usage of something that is instrumented. That said, if there is an actual LlamaIndex instrumentation maybe we should add that. Not sure if there is or if the only out-of-the-box thing available is the LLM instrumentation.

Reply:
I've added that one:

<img width="876" alt="Screenshot 2024-12-23 at 11 42 04" src="https://github.com/user-attachments/assets/149b9c4c-db38-4277-a8b6-3f77127ca0bd" />